### ISSUE TITLE
False positive: `print-statement` E1601 is true for print function

### ISSUE THREAD

**Issue by darakian** [PyCQA/pylint#2979]
### Steps to reproduce
1. Create python file with `print("Hello world")`
2. Run `pylint --py3k hello.py`
3. Receive error

### Current behavior
Usage of the python 3 print function triggers pylints print statement checker

### Expected behavior
No error

### pylint --version output
```
(venv) bash-3.2$ pylint --version
pylint 1.9.4,
astroid 1.6.6
Python 2.7.16 (default, Mar  4 2019, 09:01:38)
[GCC 4.2.1 Compatible Apple LLVM 10.0.0 (clang-1000.11.45.5)]
```

**Comment by PCManticore** [comment]
You need to use `from __future__ import print_function` for `print` to actually be a function and not a statement.

**Comment by darakian** [comment]
I'm not sure I see the difference. Perhaps I'm missing this due to my own usage of pylint, but my expectation of the lint rule is to ensure that my code is python 3 print compatible. If `print("foo")` maps to the statement in python 2 this seems like a non-issue to me.

**Comment by PCManticore** [comment]
Well there is a difference. If your `print` suddenly has more than one argument, you'll print a tuple instead of actually printing the values as expected:
```
⋊> ~ python -c "print(1, 2)"                                                                                                                                                                                                                                            12:31:39
(1, 2)
⋊> ~ pyenv shell 3.6.5                                                                                                                                                                                                                                                  12:31:44
⋊> ~ python -c "print(1, 2)"                                                                                                                                                                                                                                            12:31:48
1 2
```

This is why `print_function` needs to be imported in order for your `print` to be a function and no longer a statement.

**Comment by darakian** [comment]
Ah I see, I didn't consider that. Thanks for the explanation!
